chain/dependencies.go

```diff
@@ -217,6 +219,10 @@ type Object interface {
 type Action interface {
 	Object
 
+	// GetActionID returns the LID for an [Action] in a [Transaction]. There may be
+	// multiple [Action]s, so we pass its index in the [Action] array along with the txID.
+	GetActionID(i uint8, txID ids.ID) codec.LID
```

Comment:
This should just be a helper (not on the action) -> no data in `Action` is used to construct this.

Reply:
also prefer `txID|index`

Reply:
Or you should comment that we do this to stay consistent with how address construction works